context.go

```diff
@@ -69,10 +69,14 @@ func tokenFromHTTPForm(r *http.Request) string {
 		return ""
 	}
 
-	return r.Form.Get("castle_request_token")
+	if tkn := r.Form.Get("castle_request_token"); tkn != "" {
+		return tkn
+	}
+
+	return r.Form.Get("castle-request-token")
```

Comment:
I don't think I get this; doesn't the return statement at the end also capture the if block above? 🤔

Reply:
Ah, one is `-` and the other is `_` 🤦

Reply:
The parameters are slightly different between the two calls.

Alternatively you could define these as `validTokenNames := []string{...}` & then loop over each checking them?